dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GitHubUtil.java

```diff
@@ -110,18 +112,18 @@ public int createPullReq(GHRepository origRepo, String branch,
             log.warn("Handling error with pull request creation... {}", e.getMessage());
             JsonElement root = JsonParser.parseString(e.getMessage());
             JsonArray errorJson = root.getAsJsonObject().get("errors").getAsJsonArray();
-            String error = "";
-            if (errorJson.get(0).getAsJsonObject().has("message")) {
-                error = errorJson.get(0).getAsJsonObject().get("message").getAsString();
-            } else {
-                // This case usually happens when the PR head has invalid data. Deleting the forked repo resolves it.
-                error = PR_INVALID_CODE;
+            String error = PR_INVALID_CODE;
+            JsonObject errorMessage = errorJson.get(0).getAsJsonObject();
+            if (errorMessage.has("message")) {
+                error = errorMessage.get("message").getAsString();
             }
             log.info("error: {}", error);
+            String finalError = error;
+            List<String> errorPrefixes = Arrays.asList(PR_INVALID_CODE, "No commits between");
             if (error.startsWith("A pull request already exists")) {
```

Comment:
Not sure why you need `finalError` instead of just using `error` but you should probably use either one or the other in both if statements.

Reply:
When I tried using the variable `error` I got the following warning on IDE
`Variable used in lambda expression should be final or effectively final`